cfpb/cfgov-refresh
# Convert rate explorer cf-icons to SVG

## Changes

- Converts rate explorer icons to SVG.

## Testing

1. Run a server and visit http://localhost:8000/owning-a-home/explore-rates/
2. Check the download icon in the upper-right of the chart.
3. Hit the "I won't buy for several months" tab to see the arrows.

## Screenshots


Before:

<img width="669" alt="Screen Shot 2019-04-29 at 5 47 25 PM" src="https://user-images.githubusercontent.com/704760/56929392-6f3f1a80-6aa7-11e9-9c6b-738c144c895b.png">

<img width="159" alt="Screen Shot 2019-04-29 at 5 52 11 PM" src="https://user-images.githubusercontent.com/704760/56929419-8120bd80-6aa7-11e9-8c86-2313a850ba4f.png">


After:
<img width="644" alt="Screen Shot 2019-04-29 at 5 48 40 PM" src="https://user-images.githubusercontent.com/704760/56929344-59c9f080-6aa7-11e9-94e5-10b5fd11ef8c.png">

<img width="159" alt="Screen Shot 2019-04-29 at 5 46 54 PM" src="https://user-images.githubusercontent.com/704760/56929352-5d5d7780-6aa7-11e9-9d83-1249cc8ed147.png">
